src/pages/[platform]/build-a-backend/data/custom-subscription/index.mdx

```diff
@@ -217,7 +217,7 @@ const schema = a.schema({
     })
     .returns(a.ref('Message'))
     .handler(a.handler.custom({ entry: './publish.js' }))
-    .authorization([a.allow.public()]),
+    .authorization(allow => [allow.publicApiKey()]),
```

Comment:
Should there be another change on line 228?

From https://github.com/aws-amplify/docs/pull/7252/files#diff-055f3927edf8d3749a150a7fe3010aaf5f503e8e27c06f8df3025b2e96576e04

![image](https://github.com/aws-amplify/docs/assets/54393192/9611fac8-6f22-4d58-b231-c0ab972c8300)

Reply:
Yes, thanks for catching!